draft-ietf-tls-rfc8446bis.md

```diff
@@ -1655,6 +1655,13 @@ Here:
 - "extension_data" contains information specific to the particular
   extension type.
 
+The contents of the "extension_data" field are typically defined by an
+extension-specific structure defined in the TLS presentation language. Unless
+otherwise specified, trailing data is forbidden. That is, senders MUST NOT
+include data after the structure in the "extension_data" field, and receivers
+MUST abort the handshake with a "decode_error" alert if there is data left
+over after parsing the structure.
```

Comment:
Could you please clarify this so that it is clear that if the receiver doesn't parse the extension data, then it doesn't have to check it? The way it is worded now seems to imply that receivers must parse every extension every time, just to comply with this.

Reply:
Fair enough. Does the new text work?